frontend/src/components/pages/CardDetails/Presentation.jsx

```diff
@@ -19,33 +19,66 @@ import AssigneesList from "../../widgets/AssigneesList";
 import ReviewersTable from "../../widgets/ReviewersTable";
 import Reviews from "./Reviews";
 
-const useStyles = makeStyles((theme) => {
-  return {
-    paper: {
-      padding: theme.spacing(1, 2, 1),
-    },
+const TableCell = withStyles({
+  root: {
+    borderBottom: "none",
+    padding: "3.24%",
+  },
+})(MuiTableCell);
 
-    commentColumn: {
-      minWidth: 300,
+const useStyles = makeStyles((theme) =>
+  createStyles({
+    root: {
+      flexGrow: 1,
     },
-
-    tableContainer: {
-      maxHeight: 200,
+    paper: {
+      padding: theme.spacing(2),
+      height: "100%",
+      textAlign: "auto",
+      color: theme.palette.text.primary,
+      boxSizing: "border-box",
+      maxHeight: 350,
     },
-    sectionPaper: {
+    gridItem: {
+      boxSizing: "border-box",
       padding: theme.spacing(1),
-      marginBottom: theme.spacing(1),
     },
-    root: {
-      margin: theme.spacing(1),
-      width: theme.spacing(40),
-      height: theme.spacing(16),
+    masonryItemFirst: {
+      boxSizing: "border-box",
+      padding: `0 ${theme.spacing(1)}px ${theme.spacing(1)}px ${theme.spacing(
+        1
+      )}px`,
+      [theme.breakpoints.up("lg")]: {
+        padding: `0px ${theme.spacing(1)}px 0px 0px`,
+      },
+      [theme.breakpoints.down("sm")]: {
+        padding: `0px ${theme.spacing(1)}px 0px 0px`,
+      },
+    },
+    masonryItemSecond: {
+      boxSizing: "border-box",
+      padding: `${theme.spacing(1)}px ${theme.spacing(1)}px 0px ${theme.spacing(
+        1
+      )}px`,
+      [theme.breakpoints.up("lg")]: {
+        padding: `0px 0px 0px ${theme.spacing(1)}px`,
+      },
+      [theme.breakpoints.down("sm")]: {
+        padding: `0px 0px 0px ${theme.spacing(1)}px`,
+      },
+    },
+    grid: {
```

Comment:
Doesn't the Grid component have a spacing prop? Why are you implementing this here? 

https://v4.mui.com/components/grid/#spacing